src/lib/jsonStatham.js

```diff
@@ -18,10 +18,9 @@ var jsonStatham = {
   setAdaptor(adaptor) { this.adaptor = adaptor },
   getAdaptor() { return (this.adaptor ? this.adaptor : new defaultAdaptor()) },
   buildRequest(path, method, data, withCredentials) {
-    var headers = {} // TODO grab special header
     var adaptor = this.getAdaptor()
     var opts = {
-      headers,
+      headers: adaptor.headers(),
```

Comment:
we should probably open this up to do an assign to some passed headers in the options. (but don't need that for this PR).